pycbc/__init__.py

```diff
@@ -86,7 +86,8 @@ def add_common_pycbc_options(parser):
                             'logging at the info level, but -vv or '
                             '--verbose --verbose provides debug logging.')
 
-def init_logging(verbose=False,
+
+def init_logging(verbose=False, default_level=0, to_file=None,
```

Comment:
This is probably the best place to start the review - the more significant change is here:
 - We add a default_level kwarg so that the code can define the default level of logging to be higher, but --verbose still adds incremental logging levels
 - We add a to_file kwarg, so that we can define logging to a file through the init_logging interface. If wanting logging to more than one file, or to both stderr and the file, we can call init_logging twice with different to_file values